plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -1066,20 +1092,7 @@ void GSRendererDX11::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sou
 
 		dev->SetupPS(m_ps_sel, &ps_cb, m_ps_ssel);
 
-		bool z = m_om_dssel.zwe;
-		bool r = m_om_bsel.wr;
-		bool g = m_om_bsel.wg;
-		bool b = m_om_bsel.wb;
-		bool a = m_om_bsel.wa;
-
-		switch(m_context->TEST.AFAIL)
-		{
-			case AFAIL_KEEP: z = r = g = b = a = false; break; // none
-			case AFAIL_FB_ONLY: z = false; break; // rgba
-			case AFAIL_ZB_ONLY: r = g = b = a = false; break; // z
-			case AFAIL_RGB_ONLY: z = a = false; break; // rgb
-			default: __assume(0);
-		}
+		bool z, r, g, b, a;
```

Comment:
The alpha test changes aren't really needed for this pr, it makes reviewing the pr harder, cleanups can be done at a later time.